src/mon/OSDMonitor.cc

```diff
@@ -2832,7 +2832,8 @@ bool OSDMonitor::preprocess_boot(MonOpRequestRef op)
     goto ignore;
   }
 
-  if (osdmap.test_flag(CEPH_OSDMAP_PGLOG_HARDLIMIT) &&
+  if (osdmap.require_osd_release >= CEPH_RELEASE_LUMINOUS &&
+      osdmap.test_flag(CEPH_OSDMAP_PGLOG_HARDLIMIT) &&
```

Comment:
not sure this is needed?  and it's also luminous, which doesn't seem quite right

Reply:
it's to distinguish from the jewel feature bit since there's none left that we could reuse without the release check in luminous - maybe worth noting in the commit message

Reply:
updated!